automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/user/SignInPage.scala

```diff
@@ -108,6 +115,13 @@ class GoogleSignInPopup(implicit webDriver: WebDriver) extends WebBrowser with W
     returnFromSignIn()
   }
 
+  private def oneAccountSignIn(email: String, password: String): Unit = {
+    emailField(id("email")).value =(email)
+    find(id("next")).get.underlying.submit()
+    pwdField(id("Passwd")).value = password
+    find(id("signIn")).get.underlying.submit()
+  }
```

Comment:
Do we need a `returnFromSignIn()` here? I don't know for sure that this new sign-in flow is in a new window but my impression from the test failure screenshots I've seen is that it is.

Reply:
seems true. I added `returnFromSignIn` in `signIn`